pkg/config/config.go

```diff
@@ -42,7 +42,8 @@ func New() Config {
 		PostgresImageStreamTag: getEnv("POSTGRES_IMAGE_STREAM_TAG", "10"),
 
 		// these are used when the image stream does not exist and created for the first time by the operator
-		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/aerogear/unifiedpush-wildfly-plain:2.2.1.Final"),
+		// UPSImageStreamInitialImage:     getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/aerogear/unifiedpush-configurable-container:2.3.0"),
+		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/unifiedpush-configurable-container:2.3.0"),
```

Comment:
```suggestion
		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/unifiedpush-configurable-container:2.3"),
```

@danielpassos Myself and @wei-lee were just talking about this, and we think maybe using the `2.3` tag might be better. That way, patch fixes in the UPS project (e.g. `2.3.1`) can be released without needing a new release of the operator. What do you think?

(the process to update to the `2.3.1` release would just be that the SRE team would do an `oc import-image ups-imagestream` during their scheduled maintenance window)

Also, could you change the default for `UPS_IMAGE_STREAM_TAG` on line 34 to be `2.3` also?

Reply:
Done!